src/lib/provable/bigint.ts

```diff
@@ -752,7 +753,7 @@ abstract class ProvableBigInt<T> {
   abstract assertEquals(a: T): void;
 }
 
-function rangeCheck(x: Field, bits: number, signed?: boolean) {
+function rangeCheckHelper(x: Field, bits: number, signed?: boolean) {
   const supportedBits = new Set([32, 48, 64, 116, 128]);
```

Comment:
Also, I just spotted this - why not simply enforce this on the type level? That would be much cleaner and easier given that we use typescript for that exact reason